packages/lib-user/src/components/UserHome/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,44 @@
+import { Box, Image } from 'grommet'
+import styled from 'styled-components'
+import { shape, string } from 'prop-types'
+
+const Relative = styled(Box)`
+  position: relative;
+`
+
+const StyledAvatar = styled(Image)`
+  width: 128px;
+  height: 128px;
```

Comment:
Once https://github.com/zooniverse/front-end-monorepo/pull/6115 is merged, I'll add some mobile styling to this area of the Dashboard.

Reply:
Done 👍